src/features/common/types/map.d.ts

```diff
@@ -9,9 +13,11 @@ import type {
   Feature,
   FeatureCollection,
   MultiPolygon,
+  Point,
   Polygon,
 } from 'geojson';
 import type { Map as MapLibreMap } from 'maplibre-gl';
+import type { InterventionProperties } from '../../projectsV2/ProjectsMap/microComponents/InterventionLayers';
 
```

Comment:
You're exporting InterventionProperties from this file. This appears to be introduced accidentally.